### PR TITLE
Update default template/static to brand as Candid

### PR DESCRIPTION
**Pull request by stgraber**
This drops the Juju logo from all pages, re-titles pages to Candid, drop some empty headers, make Copyright data more consistent, drop the favicon, fix some indents and make the templates match closer.

Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

### CODE REVIEW

**Comment by albertodonato** [comment]
FWIW every app should have a favicon so that clients don't keep asking for it and get 404 back

**Comment by stgraber** [comment]
Someone should come up with a Candid logo that we can use as favicon :)

**Comment by carlaberkers** [comment]
We should talk about whether its possible to not have this page altogether.
If we MUST have the page it should have the logo of the app you come from (Juju if you come from Juju, LXD if you come from LXD etc).
If we CANT have the logo of the app and we MUST have a favicon, lets put the Canonical logo and wordmark for now, and have a conversation on the above in Cape Town.

Sounds good @anthonydillon @stgraber @mhilton @albertodonato?

**Comment by albertodonato** [comment]
We need this page, it has the same function as the login page of the Ubuntu SSO you get redirected to if you try to access a service that requires it.

It shouldn't really be service-specific, as it's a service on its own (the Canonical authentication service). I'm not sure there's a way for Candid to show conditional content based on which service the user is authenticating for, but @mhilton would know more.

File: templates/login-form [hunk 1]
```diff
@@ -26,31 +26,26 @@ if (/*@cc_on!@*/false) {
...
-                    </div>
-                    <div class="login__message">
-                    </div>
-                </div>
-            </div>
+      <div id="login-container">
+        <div class="login">
+          <div class="login__full-form">
+            <div class="login__env-name">Login</div>{{if .Error}}
+            <div class="login__failure-message" style="">{{.Error}}</div>{{end}}
```
Comment (anthonydillon) [inline templates/login-form]: Please remove the empty style attr

**Comment by carlaberkers** [comment]
We met with Martin and Uros this morning and realised it needs a bit of thinking as all logical solutions have their challenges:
- Juju branding doesn't apply to other services like RBAC
- no branding is really bad, gives users no confidence they are in the right place
- Ubuntu branding is technically less appropriate than Canonical Branding
- Canonical branding is a bit funny as users might be sent to Ubuntu SSO from here (or Microsoft or Google, in any case, three different brands in a row)

We'll land it with branding or favicon for now, make a future proposal, and then discuss and agree next steps in Cape Town.

File: templates/login-form (hunk 1 above)
Comment (stgraber) [reply templates/login-form]: Done

**Comment by carlaberkers** [comment]
I approve :-)

**Comment by anthonydillon** [comment]
@mhilton is the ready to be merged

**Comment by albertodonato** [comment]
> * Canonical branding is a bit funny as users might be sent to Ubuntu SSO from here (or Microsoft or Google, in any case, three different brands in a row)

@carlaberkers if Candid is configured with USSO or other external web-based providers this page won't be used as Candid will redirect directly to the provider login page.

**Pull request merged**